schedule/yast/firstboot/yast2_firstboot_custom-sle.yaml

```diff
@@ -0,0 +1,28 @@
+---
+name:           yast2_fistboot_custom
+description:    >
+    Variant of YaST2 Firstboot module using custom control file. Allows us to test different scenarios,
+    and to see if the yast2-firstboot package remains stable whith an old, potentially deprecated control file.
+vars:
+    YAST2_FIRSTBOOT_USERNAME: firstbootuser
```

Comment:
As we start using test data, I guess it's time to also get rid of this variable both in configuration and validation part. It's out of scope here, so could you please file a ticket? Or I can do that too, unless you are fine to do as a part of this change. WDYT?

Reply:
Do you mean we should move this to test data instead of variable ? I guess I can submit a ticket or even do it.

Reply:
Yep, so that we have everything in one place basically. Whatever works better for you, there is no issue to address this later ;)

Reply:
I'll make a ticket